consensus/statehash.go

```diff
@@ -18,9 +18,9 @@ func (s *State) sortedUtxoSet() (sortedOutputs []Output) {
 	for _, utxoString := range unspentOutputStrings {
 		var outputID OutputID
 		copy(outputID[:], utxoString)
-		output, err := s.output(outputID)
-		if err != nil {
-			panic(err)
+		output, exists := s.output(outputID)
+		if !exists {
+			panic("output doesn't exist?")
```

Comment:
right, first I had to look at everyone who was calling `sortedUtxoSet`, and so I grepped and saw that it was only called from `hashState`, but the calls weren't wrapped in `DEBUG`. So then I had to grep for all the calls to `hashState` to verify that *those* calls were wrapped in `DEBUG`s.